README.md

```diff
@@ -1,93 +1,11 @@
-CustomGauge
+CustomGauge with light effect animation
```

Comment:
Please just add new section about light effect animation at the bottom of README instead of removing its content and renaming lib